spec/app/models/metasploit_data_models/automatic_exploitation/match_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe MetasploitDataModels::AutomaticExploitation::Match do
```

Comment:
Missing the following test sections:

* Mass Assignment Security
* Database (columns and indexes)